components/ws-daemon/pkg/content/initializer.go

```diff
@@ -41,6 +42,12 @@ type RunInitializerOpts struct {
 	OWI map[string]interface{}
 }
 
+// errors to be tested with errors.Is
+var (
+	// cannot find snapshot
+	ErrCannotFindSnapshot = errors.New("cannot find snapshot")
```

Comment:
`ErrCannotFindSnapshot` does not necessarily be public (i.e. start with an uppercase letter), it's only used within the same package.

Reply:
ah!